open_academy/models/partner.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+    instructor = fields.Boolean()
+    teacher_type = fields.Selection(
```

Comment:
Leave a blank space to separate the fields
```suggestion
    _inherit = 'res.partner'
    
    instructor = fields.Boolean()
    teacher_type = fields.Selection(
```

Reply:
But this a violation to flake8 and pylynt rules, isn't it?

Reply:
Did the linter mark this as an error? If it did, could you send a screenshot please?

Reply:
I was wrong, the change is done